specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/EnterpriseKnowledgeGraphSwagger.json

```diff
@@ -0,0 +1,2001 @@
+            "name": "top",
+            "description": "Maximum number of entity documents to return",
+            "type": "integer",
+            "format": "int32",
+            "default": 10
+          },
+          {
+            "in": "query",
+            "description": "Type of entity to return, e.g. project or employee",
+            "name": "filter",
+            "type": "string",
+            "default": ""
+          },
+          {
+            "in": "query",
+            "name": "includeEntityDocument",
+            "description": "1 to include entityDocument in response body, 0 not to include",
+            "type": "integer",
+            "format": "int32",
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "enableQueryRewrite",
+            "description": "1 to rewrite query, such as removing stop words, etc., 0 to not",
+            "type": "integer",
+            "format": "int32",
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "debug",
+            "description": "1 to show debug info like index, etc., 0 to not",
+            "type": "integer",
+            "format": "int32",
+            "default": 0
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DocumentSearchResult"
+            }
+          },
+          "default": {
+            "schema": {
+              "$ref": "#/definitions/NotFoundResult"
+            }
+          }
+        }
+      },
+      "/search/suggestions": {
+        "get": {
+          "tags": [
+            "Search"
+          ],
+          "summary": "Provide suggestions from query",
+          "operationId": "GetQuerySuggestion",
+          "produces": [
+            "application/json"
+          ],
+          "parameters": [
+            {
+              "in": "query",
+              "name": "q",
+              "description": "Natural language query",
+              "type": "string"
+            },
+            {
+              "in": "query",
+              "name": "top",
+              "description": "Maximum number of suggestions to return",
+              "type": "integer",
+              "format": "int32",
+              "default": 10
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "Success",
+              "schema": {
+                "uniqueItems": false,
+                "type": "array",
+                "description": "A list of suggestion terms",
+                "items": {
+                  "type": "string"
+                }
+              }
+            },
+            "default": {
+              "schema": {
+                "$ref": "#/definitions/NotFoundResult"
+              }
+            }
+          }
+        }
+      }
+    },
+    "definitions": {
+      "ConverseRequest": {
```

Comment:
There are missing descriptions for almost all attributes. Understanding what values are allowed for attributes like `entityType` will be very challenging for users of the API.